lighthouse-core/gather/gatherers/trace-elements.js

```diff
@@ -137,6 +137,26 @@ class TraceElements extends Gatherer {
     return topFive;
   }
 
+  /**
+   * Find the node ids of elements which are animated using the Animation trace events.
+   * @param {Array<LH.TraceEvent>} mainThreadEvents
+   * @return {Array<TraceElementData>}
+   */
+  static getAnimatedElements(mainThreadEvents) {
+    const animatedElementIds = new Set(mainThreadEvents
+      .filter(e => e.name === 'Animation' && e.ph === 'b')
+      .map(e => this.getNodeIDFromTraceEvent(e)));
+
+    /** @type Array<TraceElementData> */
+    const animatedElementData = [];
+    for (const nodeId of animatedElementIds) {
+      if (nodeId !== undefined) {
```

Comment:
FWIW `node_id`, if present and a number, will always be > 0. https://source.chromium.org/chromium/chromium/src/+/master:components/ui_devtools/ui_element.cc;l=133;drc=0ee4438ca83448bca93cb11bc3012856d29303dd;bpv=0;bpt=1

so could either:

* tack on a filter(Boolean) to the end of setting `animatedElementIds`.
* or use `nodeId && animatedElementData.push({nodeId})` in the loop